src/main/pages/setup-kubernetes/kubernetes-single-user.adoc

```diff
@@ -7,28 +7,96 @@ permalink: kubernetes-single-user.html
 folder: setup-kubernetes
 ---
 
-[id="supported-kuberentes-flavors-and-versions"]
-== Supported Kubernetes flavors and versions
-
-The single-user deployment of Che to Kubernetes is tested only on Minikube v0.25 with the *VirtualBox* and *kvm2* VM providers.
+This section walks you through single-user deployment of Che on Kubernetes.
 
 *Prerequisites*
 
-* `bash`
-* `Kubernetes` installation with:
-** Enabled DNS discovery.
-** Configured Ingress controller. You can start the Ingress controller on Minikube using the `minikube addons enable ingress` command.
+* A Kubernetes cluster with at least 4 GB RAM and RBAC:
+** For Minikube 0.26.0 and higher, use the following command:
++
+----
+minikube start --cpus 2 --memory 4096 --extra-config=apiserver.authorization-mode=RBAC
+----
++
+** For Minikube 0.25.2 and lower, use the following command:
++
+----
+minikube start --cpus 2 --memory 4096 --extra-config=apiserver.Authorization.Mode=RBAC
+----
++
+* To deploy role binding, use the following command:
++
+----
+kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default
+----
++
+* Install the https://github.com/kubernetes/helm/blob/master/docs/install.md[Helm] CLI. HELM is the package manager for Kubernetes.
+* Set your default Kubernetes context (this is required to use Helm). In Minikube, this is set automatically. Otherwise, modify the `KUBECONFIG` environment variable and run the following coammnd:
++
+----
+kubectl config use-context <my-context>
+----
++
+* Install tiller, the Helm server, on your cluster.
+.. Create a https://github.com/kubernetes/helm/blob/master/docs/rbac.md[tiller serviceAccount].
++
+----
+kubectl create serviceaccount tiller --namespace kube-system
+----
++
+.. Bind it to the `cluster-admin` role.
++
+----
+kubectl apply -f ./tiller-rbac.yaml
+----
++
+.. Install `tiller`.
++
+----
+helm init --service-account tiller
+----
++
+* Ensure that you have an NGINX-based Ingress controller. This is the default Ingress controller on Minikube. To start it, use the `minikube addons enable ingress` command.
```

Comment:
Actually, Che is able to work on Kubernetes if any Ingress controller is there.

Reply:
ok